### ISSUE TITLE
unsync first and end point

### ISSUE THREAD

**Issue by p-k-l** [sjaehn/BShapr#15]
Hello @sjaehn 
please add option for unsync [this](https://imgur.com/tpxdASG) points position (dont move together)
i use BShapr instead flstudio grossbeat but this point dose not allow me to create sloping line look like>> \

**Comment by sjaehn** [comment]
Thanks to mention. I understand your point. If you need a slope, simply draw that slope just before the end and then a line back to the baseline.

Background information: B.Shapr is a loop-based plugin. If the start and the end point do not have the same value, this would result in a hard jump. If this is not handled by the plugin, hard jumps may result in nasty clicks. These clicks may destroy your ears or speakers in the worst case. Thus, loop-based plugins with start != end have internally to draw a line from the end point to the start point to prevent this problem. You see that you end up at the same point - if *you* draw the line or the *plugin* creates an invisible line.

**Comment by p-k-l** [comment]
@sjaehn thanks for replay 
in this method when use doppler delay add a click (sound) in slope point position

**Comment by sjaehn** [comment]
Right, the doppler delay may produce clicks. But this is not a bug.

**Comment by p-k-l** [comment]
@sjaehn 
ok 
plz add time saphper preset/function looklike [cableguys time shaper](https://www.cableguys.com/timeshaper.html)
and plugin resize button b-shapr have a big gui 
tnx

**Comment by sjaehn** [comment]
> and plugin resize button b-shapr have a big gui

The GUI is freely resizable (as long as the host (= the DAW) supports the resize feature). Like all other B.Music plugins. Try to resize by dragging the bottom right corner. Or simply maximize the window.

**Comment by sjaehn** [comment]
> plz add time saphper preset/function looklike [cableguys time shaper](https://www.cableguys.com/timeshaper.html)

Good point. I like the way to store and preview shapes in Cableguys Shaperbox plugins. However, LV2 has a different way to store presets. It's the hosts responsibility to store/restore them. This makes it easy for the plugins as the plugin doesn't need to care about it. But it makes it really complicataed if the plugin wants to take care.

The way I see would be not to store the full preset but only a single shape without any control port data.
